src/Extension/ExtensionManager.php

```diff
@@ -318,6 +318,6 @@ public function isEnabled($extension)
      */
     protected function getExtensionsDir()
     {
-        return public_path('vendor');
+        return base_path('vendor');
```

Comment:
I guess you mean replacing `base_path('vendor')`by `$this->app->basePath().'vendor'`?